editor-basics/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "editor-howto-gallery",
+  "version": "0.1.0",
+  "productName": "Minecraft Editor How-to Gallery",
+  "description": "Minecraft Editor How-to Gallery",
+  "private": true,
+  "devDependencies": {
+    "@minecraft/core-build-tasks": "^1.1.3",
+    "eslint-plugin-minecraft-linting": "^1.2.1",
+    "source-map": "^0.7.4",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
+  },
+  "scripts": {
+    "lint": "just-scripts lint",
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "local-deploy": "just-scripts local-deploy",
+    "buildsnippets": "just-scripts buildSnippets",
+    "mcaddon": "just-scripts mcaddon",
+    "enablemcloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-1958404141-86561845-1752920682-3514627264-368642714-62675701-733520436",
+    "enablemcpreviewloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-424268864-5579737-879501358-346833251-474568803-887069379-4040235476"
+  },
+  "dependencies": {
+    "@minecraft/math": "^1.1.0",
+    "@minecraft/server": "^1.12.0-beta.1.21.0-preview.24",
```

Comment:
Should the editor basics and editor script box be moved to the same spot as server-admin/gametest/etc samples because editor is still beta?

Reply:
I think the idea was to just keep them peer folders at the root. At some point I might create a separate how-to-gallery for experimental APIs (howto-gallery-experimental) to re-introduce some gametest and server-net stuff.. and that'd be a peer of the howto-gallery folder (maybe with some cool gametest stuff thrown in)